src/aux-server/aux-web/aux-player/PlayerHome/PlayerHome.vue

```diff
@@ -118,7 +125,6 @@
                     @click="
                         executeBiosOption(biosSelection, recordSelection, instSelection, joinCode)
                     "
-                    :disabled="!canLoad"
```

Comment:
@TroyceGowdy This looks like an accidental removal. The button should still be disabled until a valid set of options have been selected